src/test/kotlin/com/linchpino/core/security/SecurityConfigTestIT.kt

```diff
@@ -0,0 +1,44 @@
+package com.linchpino.core.security
+
+import com.linchpino.core.PostgresContainerConfig
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.header
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import org.springframework.transaction.annotation.Transactional
+
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@Transactional // Ensure rollback after each test
+@Import(PostgresContainerConfig::class)
+class SecurityConfigTestIT {
+
+    @Autowired
+    private lateinit var mockMvc: MockMvc
+
+    @Test
+    fun `test CORS with fake origin header`() {
+
+        this.mockMvc
```

Comment:
Line 30 does nothing and should be removed

Reply:
Done